cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Context.java

```diff
@@ -48,13 +49,21 @@ public Context(CgmesModel cgmes, Config config, Network network) {
         // based on existing node-breaker info
         nodeBreaker = cgmes.isNodeBreaker() && config.useNodeBreaker();
 
+        if (config.getMinimumValidationLevel() == ValidationLevel.EQUIPMENT) {
+            flowsDefaultValue = Double.NaN;
```

Comment:
Maybe add as comment that it means active and reactive power default value. The `fixSsh` could be something more explicit as `forceValidationLevelToSteadyStateHypothesis`.